src/hooks/use-is-key-pressed.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useRef, useEffect } from "react";
+
+export const useIsKeyPressed = key => {
+  const [isPressed, setIsPressed] = useState(false);
+  const keyRef = useRef(key);
+  useEffect(() => {
+    const onKeyDown = ev => {
+      if (ev.key === keyRef.current) setIsPressed(true);
+    };
+    const onKeyUp = ev => {
+      if (ev.key === keyRef.current) setIsPressed(false);
+    };
+    window.addEventListener("keydown", onKeyDown);
+    window.addEventListener("keyup", onKeyUp);
```

Comment:
use-is-key-pressed.js
Interesting react does not have key press events built in. Or is it because we are trying to capture everything.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event

keypress is deprecated

Yes I wanna catch global events and not bind them to a single element.